tests/org.eclipse.n4js.n4jsx.spec.ui.tests/xpectTests/compile/replacer.n4js

```diff
@@ -13,12 +13,7 @@
 // See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
 export public function replacer(key: string, value: any): any {
 	if (value instanceof Function) {
-		//Extract the function name out of the function string
-		//Note: this does not work for anonymous functions! But this is just a test.
-		let fnName = (value as Function).toString();
-		fnName = fnName.substring('function '.length);
-		fnName = fnName.substring(0, fnName.indexOf('('));
-		return fnName.trim()? fnName.trim() : "function" ;
+		return "function";
```

Comment:
Why did you change this implementation?

Reply:
The original function tries to extract the function name but it does not work correctly. Also for the test purpose, `function` is all what we need.